packages/compass-preferences-model/src/renderer-ipc.ts

```diff
@@ -29,7 +30,13 @@ export const makePreferencesIpc = (ipc: typeof hadronIpc) => ({
         'compass:get-configurable-user-preferences'
       );
     }
-    return Promise.resolve({} as UserPreferences);
+    return Promise.resolve({} as UserConfigurablePreferences);
+  },
+  getPreferenceStates(): Promise<PreferenceStateInformation> {
```

Comment:
What is getPreferenceStates used for?

Reply:
Mh yeah, I think all of these methods could use doc comments. Added in a0f5bc0baa07a54f5ca5cb14f57f15e904ecee77!